### DISCUSSION TITLE
2011/02/03/%E8%B1%86%E8%B1%86%E4%B8%A4%E4%B8%AA%E6%9C%88

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#312] [Announcements]
# 2011/02/03/%E8%B1%86%E8%B1%86%E4%B8%A4%E4%B8%AA%E6%9C%88

大年初一，豆豆出生整两个月。不过按照中国传统的虚岁算法，豆豆今天都两岁了。

https://blog.qizhen.xyz/2011/02/03/%E8%B1%86%E8%B1%86%E4%B8%A4%E4%B8%AA%E6%9C%88

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Huajun
2011年02月4日 02:02 编辑
日子过得真快，一晃我也为人叔父了。虽尚未见过小侄儿一面，却已充满期待。
不知道在他们的眼中，这个世界是什么样的，以为着什么。也许今后的十几年抑或几十年，他也会思索这样的问题吧。
也许他的世界就是在一个又一个条件反射基础上的熟练、思考、应用，也许是一个又一个哎与哎的回应和沟通。
我们想要进入他的世界，了解他的想法的同时，他是不是也在试图进入我们的世界，了解我们的想法呢？

回复
